manifests/init.pp

```diff
@@ -345,6 +347,7 @@
   Variant[Undef,String] $kubernetes_ports = undef,
 
   # REST API Settings
+  Optional[String] $ciphers = undef,
```

Comment:
Correction from other comment, this needs to match the other patterns for rest API items
```suggestion
  Optional[String] $restapi_ciphers = undef,
```